### DISCUSSION TITLE
Deadline không bao giờ tha chúng ta :))) nên đây là phân công câu hỏi tuần 10

### DISCUSSION THREAD

**Discussion by thaopham2501** [baonhu1304/Lop-10A2-nhom-1#18] [General]
Tuần trước hơi gãy xíu nên tuần này lấy công chuộc tội nha :)))

**Comment by letu25082006** [comment]
nhìn cái tittle chầm cãm qớ

**Reply by thaopham2501** [reply]
tại vì chầm cãm thiệc :(

**Comment by letu25082006** [comment]
ủa sao thầy k cho 5 câu chia đều mà thích cho chia lẻ lẻ quá ha

**Reply by thaopham2501** [reply]
tại thầy thích nhìn hs đau khộ chứ sao :))))

**Comment by letu25082006** [comment]
phân công đi mng

**Reply by Lenga10A2** [reply]
5 người mà 6 câu á hả

**Reply by Lenga10A2** [reply]
thầy cho ngộ ghê á

**Comment by thaopham2501** [comment]
ok 6 câu sẽ có một người làm 2 câu :)) ai 2 câu xung phong đi

**Reply by letu25082006** [reply]
nhường

**Reply by letu25082006** [reply]
nhường Thảo đó=))

**Reply by Lenga10A2** [reply]
rồi ok t câu 1 bài 7 vs câu 1 bài 8 cho

**Reply by letu25082006** [reply]
ghê lìn

**Reply by Lenga10A2** [reply]
hihi tại 2 câu đầu lúc nào cx dễ

**Reply by letu25082006** [reply]
khó chắc t cừi xĩu

**Reply by Lenga10A2** [reply]
đừng như v mà :(((

**Reply by baonhu1304** [reply]
uh câu 1 bài 8 khum dễ đou

**Comment by letu25082006** [comment]
t câu 3 b7 đc k mng

**Comment by baonhu1304** [comment]
Toai câu 2 bài 7 nhoa

**Comment by baonhu1304** [comment]
Câu 1 bài 7 và câu 2 bài 8 đều có trong SGK nha. Nên ai làm 2 câu thì làm 2 câu này ii

**Reply by letu25082006** [reply]
ủa z kêu Nga đổi qua kìa

**Reply by baonhu1304** [reply]
@Lenga10A2 đổi i cưng

**Reply by Lenga10A2** [reply]
ờ ok v để t đổi cho

**Reply by letu25082006** [reply]
Nga tự nguyện làm 2c á

**Reply by Lenga10A2** [reply]
okok

**Comment by NguyenNgocNhuY** [comment]
tui câu 2 bài 8 đ k mn

**Reply by letu25082006** [reply]
uki luôn

**Reply by letu25082006** [reply]
ủa khoan Ý ơi

**Reply by letu25082006** [reply]
câu 2 Nga làm r á

**Reply by NguyenNgocNhuY** [reply]
v tui làm câu 1 bài 8 đc k mn

**Reply by Lenga10A2** [reply]
đúng ùi để tui làm 2 câu cho

**Reply by NguyenNgocNhuY** [reply]
ừ v Nga làm câu 1 b7 câu 2 bài 8 còn tui câu 1 b8 nhen

**Comment by baonhu1304** [comment]
@thaopham2501 gòi câu 3 bài 8 nghen

**Reply by letu25082006** [reply]
đặc biệt dành riêng cho @thaopham2501

**Reply by thaopham2501** [reply]
ác quá để t làm câu khó dzãy '-'

**Reply by baonhu1304** [reply]
Ai biểu lúc ngừi ta đang bàn, m đi ăn cơm ;))))

**Reply by thaopham2501** [reply]
ai biểu tụi bây bàn lúc t đi ăn cơm :(

**Comment by letu25082006** [comment]
alo @Lenga10A2 ơi

**Reply by Lenga10A2** [reply]
Đây đây
